apps/heureka/src/components/Services/utils.ts

```diff
@@ -106,7 +106,7 @@ export const getNormalizedServicesResponse = (data: GetServicesQuery | undefined
  * @returns A string representation of the error message, or a fallback message
  * if no specific error is found.
  */
-export const getNormalizedError = (error?: ApolloError) => {
+export const getNormalizedError = (error?: { networkError?: any; message?: string }) => {
```

Comment:
We should avoid any. I would suggest to use unknown.

```
unknown is safer because it forces you to perform type checks before using the value, preventing accidental runtime errors. any skips all type checking.
```